db/schema.rb

```diff
@@ -1283,7 +1283,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
```

Comment:
```suggestion
    t.index ["username"], name: "index_users_on_username", unique: true
    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Reply:
Seeing if we can merge the suggested change in-place without needing a fast follow